sunpy/io/file_tools.py

```diff
@@ -45,14 +44,31 @@ def __getitem__(self, key):
         return val
 
 
-# Map the readers
 _readers = Readers({
     'fits': fits,
     'jp2': jp2,
     'ana': ana
 })
 
 
+def _read(filepath, function_name, filetype=None, **kwargs):
```

Comment:
This needs a docstring with a short description and a listing of the arguments and their types.